x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -344,6 +346,29 @@ export const expandFirstTimelineEventDetails = () => {
   cy.get(TOGGLE_TIMELINE_EXPAND_EVENT).first().click({ force: true });
 };
 
+export const saveTimeline = () => {
+  cy.get(TIMELINE_SAVE_MODAL_OPEN_BUTTON).first().click();
+
+  cy.get('[data-test-subj="save-timeline-modal"]').within(() => {
+    cy.get('[data-test-subj="progress-bar"]').should('not.exist');
+    cy.get(TIMELINE_TITLE_INPUT).should('not.be.disabled');
+
+    cy.get(TIMELINE_TITLE_INPUT)
+      .invoke('val')
+      .then((value) => {
+        if (!value) {
```

Comment:
We should avoid having conditionals inside tasks. What is the reason for adding this one?

Reply:
@MadameSheema If no title has been set for the timeline, we would not be able to save it. This just makes sure to set a title before saving. It allows us to use `saveTimeline()` in our tests without passing a title attribute. At the same time, this code makes sure not to overwrite a previous title so that all previous assertions still work.

I can add a comment here, explaining why there is a conditional.

Reply:
@MadameSheema As agreed, I cleaned up the task here: https://github.com/elastic/kibana/pull/169239/commits/062915a65b0ad8a44afc095f60c5bec9f543aaf2